typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
 
 
 # get the directory path to this file; 
-LIBS_DIR = os.path.dirname(os.path.abspath(__file__))
-PLUGIN_DIR = os.path.dirname(LIBS_DIR)
-PACKAGES_DIR = os.path.dirname(PLUGIN_DIR)
+LIBS_DIR = dirname(dirname(os.path.abspath(__file__)))
+PLUGIN_DIR = dirname(LIBS_DIR)
+PACKAGES_DIR = dirname(PLUGIN_DIR)
```

Comment:
We use it mainly to load resources, like the temporary files for reloading and html template files for tooltips. I think the pattern you mentioned is ideal for retrieving the sublime package path, but we do need to get the plugin's path also, because we load binary sources from within the plugin folder and the path has to have a format like ``Packages/[PLUGIN_NAME]/icons``.